protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/distribution/CommandDistributionRecord.java

```diff
@@ -34,6 +34,10 @@ public final class CommandDistributionRecord extends UnifiedRecordValue
     RECORDS_BY_TYPE.put(ValueType.DEPLOYMENT, DeploymentRecord::new);
   }
 
+  // NOTE! When adding a new property here it must also be added to the ProtocolFactory! This class
+  // contains a randomizer implementation which is used to generate a random
+  // CommandDistributionRecord. The new property must be added there. Without it we won't generate a
+  // complete record.
```

Comment:
🔧 I think this is one of those few occasions where a block comment might be more fitting. Just to highlight this comment.